plugins/utils/nix-develop.nix

```diff
@@ -0,0 +1,41 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.nix-develop;
+  helpers = import ../helpers.nix {inherit lib;};
+in {
+  options.plugins.nix-develop =
+    helpers.extraOptionsOptions
+    // {
+      enable = mkEnableOption "nix-develop.nvim";
+
+      package = helpers.mkPackageOption "nix-develop.nvim" pkgs.vimPlugins.nix-develop-nvim;
+
+      ignoredVariables = mkOption {
+        type = types.attrsOf types.bool;
```

Comment:
you need to provide a `default = {}` for both fields. Else nix will throw an error if enable = true  and the variables are unset by the user.

Reply:
ahh yeh!